test-it/results/ComplexMathFunctions.vtr

```diff
@@ -1,9 +1,12 @@
 ReOrImToComplex 2.000000 0.500000 => (2 0.5)
```

Comment:
There are already unique strings at the beginning of each line identifying the prim and either an a or c suffix differentiating array or cluster cases. 
Adding numbers to each test case I think is a bad idea in the tests themselves because it complicates adding cases in the middle near other similar tests and is hard to maintain. 
However, along the same lines, I think the test DRIVER should output line numbers. That would solve this problem and let us update all tests easily at once.